### DISCUSSION TITLE
API for project creation...

### DISCUSSION THREAD

**Discussion by tobiasoberrauch** [supabase/supabase#4728] [Questions] [answered]
Is there a way to create a new project (like in the attached screenshot) with an API call?
If so, it would also be possible to perform the actions (like adding External OAuth Providers, change tables,...) in the admin interface than via the API?
![image](https://user-images.githubusercontent.com/202791/147665711-c012ae30-11be-49d1-9f92-a930eb50f1a7.png)

**Comment by soedirgo** [comment]
This is not available at the moment, but we're currently working on this :)

**Reply by tobiasoberrauch** [reply]
Thank you. I really like your project and I try to contribute in future :)

**Reply by kiwicopple** [reply]
This is now supported 🎉 

https://supabase.com/docs/reference/api/usage#create-a-project

**Comment by kiwicopple** [comment, marked as answer]
This is now supported 🎉

https://supabase.com/docs/reference/api/usage#create-a-project

**Reply by tobiasoberrauch** [reply]
Brilliant. I was on maternity leave and now I'm back.
Thank you. I'll try
